static/components/Map.js

```diff
@@ -79,7 +79,7 @@ const Map = (props) => {
           onClick={(marker)=>{
             console.log("selecting place in marker", marker)
             setSelectedPlace(place)
-            scrollToPlaceResult(place['Facility ID'])
+            scrollToItemRef.current[index].scrollIntoView({ behavior: "smooth" })
```

Comment:
how do u know the right facility associates with the right index? What happened to using facilityID?

Reply:
because they use the same data to map in both marker in Map.js
` const markers = placesData.results && placesData.results.length > 0 && placesData.results.map((place, index)=>{`
and placeTiles in PlaceResults.js
` const placeTiles = (placesData.results && placesData.results.length) > 0 ? placesData.results.map((place, i)=>{`

I thought this logic made sense so I didn't use facilityID